### DISCUSSION TITLE
Variational Graph Auto-Encoders?

### DISCUSSION THREAD

**Discussion by GabbySuwichaya** [pyg-team/pytorch_geometric#2760] [Q&A]
Hi! I would like to ask if you have any suggestions for PyTorch packages for Variational Graph Auto-Encoders? Or are you intend to do it? I tried to find a good reference, but it seems that the official was based on TensorFlow...  :)

**Comment by wsad1** [comment]
Check out the [GAE](https://pytorch-geometric.readthedocs.io/en/latest/modules/nn.html#torch_geometric.nn.models.GAE) module in pytorch-geometric.
